tests/ui/issues/issue-6892.rs

```diff
@@ -14,21 +14,25 @@ static mut NUM_DROPS: usize = 0;
 impl Drop for Foo {
     fn drop(&mut self) {
         unsafe { NUM_DROPS += 1; }
+        //~^ WARN creating a mutable reference to mutable static is discouraged [static_mut_refs]
```

Comment:
Since this test is unrelated, let's fix (or `allow(..)` and mark `FIXME`) this and other similar uses instead.